repository-data/webfiles/src/main/resources/site/freemarker/nationalindicatorlibrary/macro/structured-text.ftl

```diff
@@ -0,0 +1,16 @@
+<#ftl output_format="HTML">
+<#include "../../include/imports.ftl">
+
+<#macro structuredText item uipath>
```

Comment:
If this is duplicate of publication system one then move it to common so we don't have duplicates

Reply:
Not necessary in nil folder - now deleted